usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -4923,7 +4928,25 @@ lxpr_read_sys_kernel_pid_max(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	lxpr_uiobuf_printf(uiobuf, "%d\n", maxpid);
 }
 
-/* ARGSUSED */
+static void
+lxpr_gen_uuid(char *uuid, size_t size)
+{
+	uint8_t r[16];
+	if (random_get_bytes(r, sizeof (r)) != 0) {
+		(void) random_get_pseudo_bytes(r, sizeof (r));
+	}
+	/* Set UUID version to 4 (random) */
+	r[6] = 0x40 | (r[6] & 0x0f);
+	/* Set UUID variant to 1 */
+	r[8] = 0x80 | (r[8] & 0x3f);
+
+	(void) snprintf(uuid, size,
+			"%02x%02x%02x%02x-%02x%02x-%02x%02x-%02x%02x"
```

Comment:
Yeah, please fix this to be tab-plus-4-spaces?  Once that's done I'm ready to IA or encourage @papertigers to IA.